### ISSUE TITLE
Feature Request: Single File 1-click Online Lookup (e.g. VirusTotal)

### ISSUE THREAD

**Issue by mchubby** [OV2/RapidCRC-Unicode#55]
Hi,

Can you add the following scenario:

1. User right-clicks a single file in explorer,
2. selects menu entry "**RapidCRC Online Lookup**",
3. Program retrieves Lookup URL
4. computes SHA256 hash, or whichever hash is requested
5. Opens or copies URL to clipboard

Options GUI defaults:
- **Lookup URL**:  text input field `https://www.virustotal.com/en/file/%SHA256/analysis/`
- **Lookup Action**: radio (•) Open  ( ) Copy to clipboard

**Comment by vatterspun** [comment]
Good idea.  Would also like to use this functionality inside the program since I use the portable version.

**Comment by vatterspun** [comment]
One update to this: any functionality along these lines would only work for individual files and should probably come with a disclaimer about submissions.  This is because RapidCRC Unicode will check 100,000 files on your computer withing flinching, but VirusTotal (as I understand it) will only accept 4 or 5 queries without some kind of database key.
